templates/common/_base/files/kubelet-log-level.yaml

```
@@ -0,0 +1,6 @@
+mode: 0644
+path: "/etc/systemd/system/kubelet.service.d/kubelog.conf"
+contents:
+  inline: |
+[Unit]
+Environment="KUBELET_LOG_LEVEL=3"
```

```suggestion
  inline: |
      [Unit]
      Environment="KUBELET_LOG_LEVEL=3"
```

Missing indentation. Otherwise, this look much cleaner :)

---

thanks. fixed :)